versions.tf

```diff
@@ -0,0 +1,9 @@
+## Terraform Requirements
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 3.5.0"

```

Comment:
Looked at versions for Cumulus, wondering if we shouldn't bump the minimum version to match up to 3.14.1